tests/func/test_import.py

```diff
@@ -27,6 +29,34 @@ def test_import(git, dvc_repo, erepo):
     assert git.git.check_ignore(dst)
 
 
+def test_import_git_file(erepo_dir, tmp_dir, dvc, scm):
+    src = "some_file"
+    dst = "some_file_imported"
+
+    erepo_dir.scm_gen({src: "hello"}, commit="add a regular file")
+
+    tmp_dir.dvc.imp(str(erepo_dir), src, dst)
+
+    assert (tmp_dir / dst).exists()
+    assert os.path.isfile(str(tmp_dir / dst))
+    assert filecmp.cmp(str(erepo_dir / src), str(tmp_dir / dst), shallow=False)
+    assert tmp_dir.scm.repo.git.check_ignore(str(tmp_dir / dst))
```

Comment:
Should only use `str()` on Path-like objects for presentation purposes, to get str file path one should use `fspath()`. Or `fspath_py35()` when passing to an util working with Path-likes in Python 3.6+.

Also, `Path` has `.is_file()` and `.is_dir()`. May also skip `.exists()` since it is covered with `is_file()` anyway.

Reply:
Done